thirdparty/rocksdb/CMakeLists.txt

```diff
@@ -216,7 +216,7 @@ if(HAVE_THREAD_LOCAL)
   add_definitions(-DROCKSDB_SUPPORT_THREAD_LOCAL)
 endif()
 
-option(FAIL_ON_WARNINGS "Treat compile warnings as errors" ON)
+option(FAIL_ON_WARNINGS "Treat compile warnings as errors" OFF)
```

Comment:
This is just a quickfix, going to be properly done in scope of:
https://github.com/apache/nifi-minifi-cpp/pull/661